src/py42/services/fileevent.py

```diff
@@ -24,9 +24,8 @@ def search(self, query):
             :class:`py42.response.Py42Response`: A response containing the query results.
         """
         try:
-            query_str = str(query)
             uri = u"/forensic-search/queryservice/api/v1/fileevent"
-            return self._connection.post(uri, data=query_str)
+            return self._connection.post(uri, json=dict(query))
```

Comment:
Since our docstring above mentions `query` could be a `FileEventQuery` or `str`, we'll need to try to parse the string into a `FileEventQuery` first if we want to pass to `json`, otherwise this is a breaking change.

Reply:
Thanks, it should now work with str queries. I added a test as well.